src/index.html

```diff
@@ -1,13 +1,60 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="/styles/style.css">
+  </head>
+
+  <body>
+    <section class="card" data-qa="card">
+
+      <img
+        src="/images/imac.jpeg"
+        alt="iMac desktop"
+        class="card__image"
+      >
+
+      <h1 class="card__title">
+        APPLE A1419 iMac 27&#34; Retina 5K Monoblock &#40;MNED2UA&#47;A&#41;
+      </h1>
```

Comment:
h1 - it is the main title for the whole page. Here would be better to use `h2`
```suggestion
      <h2 class="card__title">
        APPLE A1419 iMac 27&#34; Retina 5K Monoblock &#40;MNED2UA&#47;A&#41;
      </h1>
```